tools/dtrace/get-ds-state.d

```diff
@@ -0,0 +1,23 @@
+/*
+ * Print a status line for a given PID.
+ * Exit after 5 seconds.
+ */
```

Comment:
question in the header comment: this prints for all PIDs rather than a given PID right?

Reply:
That was copy cruft from where I copied the script from.  I've fixed the comment to address reality.